mava/systems/ppo/sebulba/ff_ippo.py

```diff
@@ -11,3 +11,811 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import copy
+import queue
+import threading
+import time
+from collections import deque
+from typing import Any, Dict, List, Tuple
+
+import chex
+import flax
+import hydra
+import jax
```

Comment:
Seeing as we just updated the jax version, I'd like to start using the tree like this:

```suggestion
import jax
from jax import tree
```

And then lower down can you call `tree.map(...)` it's a bit cleaner imo